README.md

```diff
@@ -52,4 +60,15 @@ To shut down the development containers, you can use:
 More details
 ------------
 
-For more details on the individual runners, check out the README files in their respective directories.
\ No newline at end of file
+For more details on the individual runners, check out the README files in their respective directories.
+
+Troubleshooting
+------------
+1. Errors saying `... unsupported option: 'target'` after running `docker-compose up --build`.
+   
+   This is most likely a problem with the docker-compose version. Simply re-install it:
+
+        pip3 uninstall docker-compose
+        pip3 install -U docker-compose
+
+    
```

Comment:
Can you add a reference to the PAT issue, and how to resolve it?

(i.e., go to settings, create a PAT at https://github.com/settings/tokens, and add it as an env variable called `GITHUB_PAT`)